x/community/testutil/main.go

```diff
@@ -30,6 +33,10 @@ func (suite *Suite) SetupTest() {
 	suite.App = tApp
 	suite.Ctx = ctx
 	suite.Keeper = tApp.GetCommunityKeeper()
+	communityPoolAddress := tApp.GetAccountKeeper().GetModuleAddress(types.ModuleAccountName)
+	// hello, greppers!
+	suite.Equal("kava17d2wax0zhjrrecvaszuyxdf5wcu5a0p4qlx3t5", communityPoolAddress.String())
```

Comment:
(I know this is copy pasted code but...can this value come from a constant that lives outside of test land?)

Reply:
this isn't copypasta, it was just me keeping the address in source control for the inevitable search. originally only added it to get the address for documentation. i left it b/c it felt useful.
the constant is calculated based off the `types.ModuleAccountName`. i don't think it makes sense to keep the string itself as a constant outside the tests.
as a test, this catches regressions in the module account name, or breaking changes in how macc addresses are determined.